### PR TITLE
Refactor: Group Complex Number Tests Into Well Named Categories.

### PR DESCRIPTION
**Pull request by hemalvarambhia**
In this PR, I groupes the test methods into well-named categories and added additional tests.

This is not the end- I will keep iterating. 

Ref: [Structure and Interpretation of Test Cases, Kevlin Henney](https://youtu.be/tWn8RA_DEic)

### CODE REVIEW

File: src/Math-Tests-Complex/PMComplexTest.class.st [hunk 1]
```diff
@@ -4,7 +4,7 @@ Class {
 	#category : #'Math-Tests-Complex'
 }
 
-{ #category : #tests }
+{ #category : #'unary operations' }
```
Comment (SergeStinckwich) [inline src/Math-Tests-Complex/PMComplexTest.class.st:7]: I would prefer 'tests - unary operations'. Some library use this pattern.

**Review by SergeStinckwich — CHANGES_REQUESTED** [review]

File: src/Math-Tests-Complex/PMComplexTest.class.st (hunk 1 above)
Comment (hemalvarambhia) [reply src/Math-Tests-Complex/PMComplexTest.class.st:7]: Cool. Is this something you'd like for the other categories ?

Comment (hemalvarambhia) [reply src/Math-Tests-Complex/PMComplexTest.class.st:7]: I'll make the change in the next commit.

**Comment by hemalvarambhia** [comment]
I have merge conflicts so I will close this and issue a new PR, taking @SergeStinckwich 's feedback into account

**Pull request closed without merging**